test/unit/modules/cli/cli.spec.ts

```diff
@@ -140,6 +140,26 @@ describe('startServer via API', () => {
       global.process = realProcess;
     });
 
+    test('should start a https server with key and cert', async (done) => {
+      const store = path.join(__dirname, 'partials/store');
+      const serverName = 'verdaccio-test';
+      const version = '1.0.0';
+      const address = 'https://www.domain.com:443';
+
+      const conf = config();
+      conf.https = {
+        key: path.resolve(__dirname, 'key.pem'),
```

Comment:
Do those certificates has expiration date?

Reply:
Thank you for your feedback, it's a very good question! It would actually expire in 30 days... I am not sure would `https.createServer` work with expired certificates. 

How about using [selfsigned](https://www.npmjs.com/package/selfsigned) to generate certificate on the fly? The code would look like
```js
// cli.spec.ts
const { private: key, cert } = selfsigned.generate();
const keyPath = path.join(os.tmpdir(), 'key.pem');
const certPath = path.join(os.tmpdir(), 'crt.pem');
fs.writeFileSync(keyPath, key);
fs.writeFileSync(certPath, cert);

const conf = config();
conf.https = {
  key: keyPath,
  cert: certPath,
};
```

Reply:
OH nice, that's is pretty handy :) ok to me.